pulser-pasqal/requirements.txt

```diff
@@ -1 +1 @@
-pasqal-sdk == 0.1.6
+pasqal-sdk == 0.1.12
```

Comment:
Do you reckon we will still need a strict version pin or could we potentially go with `pasqal-sdk >= 0.1.12` or `pasqal-sdk ~= 0.1.12` ?

Reply:
good point, I'll discuss it tomorrow and let you know before merging.

Perhaps we could just add a simple test using pulser-pasqal to the cloud-e2e tests, that way the cloud team will be aware and can raise the issue?

Reply:
I think we should stick to a strict version pin, we recently overhauled the authentication system and I predict we'll have to rework a few things which might break compatibility with pulser-pasqal.

In the long run I definitely think we should have no breaking changes introduced and hence this could target the latest pasqal-sdk version. But I'm worried it's a bit too early for that

Reply:
My only issue is that PulserPasqal is going to be running "in production", it is how some quantum libraries connect to the cloud apparently.  We can perhaps also change that. But I'm not sure if that would be taking a step backwards considering future plans. For new external users the most natural thing might become to use the PulserPasqal module too. 

In a production system we'll of course also pin the cloud-sdk version down to whatever is needed to support the respective environment, but it would be nice for these versions to be valid, and not something we have to forcefully override. 

We could run some simple test in our end to end tests using PulserPasqal, which should allow us to detect changes more easily.
If we then also introduce the dev releases on cloud-sdk we might be able to keep our released packages in sync.

I admit it is a bit annoying and will cause some overhead, but at the same time so does the current way of doing things. At least this way we'll always be on top of things?

I would suggest that we for now add it as a `~=`, dependency, so we can merge this now. When there is a new release shortly on the cloud-sdk, but that won't make any changes to the interface we'll get that one for free, and the next time the cloud-sdk has to be updated externally facing we can bump the minor to `0.2.x`?

@bejito Any thoughts?

Reply:
Adding pulser-pasqal to our e2e tests battery sound like a good idea!

Not sure I understand what you imply in the two first paragraphs !

Reply:
"late" night rambling. 

We're connecting to "the cloud"  via PulserPasqal for the upcoming release of quantum discovery module 2.

Reply:
Ah right and you're saying that this is going to be painful becase to update the version of the SDK we'll need to release a new version of pulser, is that correct ?

Reply:
correct, this dependency will be manifested inside the PCS infra.

Reply:
+1 for the `~=` option, if possible.

Reply:
Yes ok let's go for this then

Reply:
Alright, I've pushed. And I've added an issue in the (internal) end-to-end tests.